backend/src/routes/api/messaging/get_groupchats.ts

```diff
@@ -0,0 +1,48 @@
+import { Request, Response } from "express";
+import GroupChats from "../../../models/Groupchat";
+import groupchat from "../../../types/groupchat";
+import Messaging from "../../../models/Messages";
+import {User} from "../../../models/User";
+import user from "../../../types/user";
+
+async function getGroupsChats(req: Request, res: Response) {
+  try {
+    let allGroupChat = [];

```

Comment:
same thing here, make sure use verifytoken and make the const groupChats = to the ids of all the groupchats that the user is in